docs/api/messages.rst

```diff
@@ -420,6 +427,23 @@ PollLayoutType
 
         The default poll layout type.
 
+MessageReferenceType
+~~~~~~~~~~~~~~~~~~~~
+
+.. class:: MessageReferenceType
+
+    Specifies the type of :class:`MessageReference`. This is used to denote
+    if a message is normal or not (e.g replying or forwarding).
```

Comment:
```suggestion
    Specifies the type of :class:`MessageReference`. This can be used to determine
    if a message is e.g. a reply or a forwarded message.
```
I think I get the idea behind "normal", but it's not clear to users immediately